### PR TITLE
MaryiaF/fix: TrackJS error this.controls.crossX.classList.replace is …

### PR DESCRIPTION
**Pull request by maryiafrantsava-deriv**
…not a function

### CODE REVIEW

File: chartiq/development/js/standard.js [hunk 1]
```diff
@@ -1127,14 +1127,26 @@ CIQ.ChartEngine.prototype.drawingClick = function (panel, x, y) {
...
-				this.controls.crossY.classList.replace(
-					"stx_crosshair_drawing",
-					"stx_crosshair"
-				);
+				if(typeof this.controls.crossX.classList === 'string'){
+					this.controls.crossX.classList.replace(
+						"stx_crosshair_drawing",
+						"stx_crosshair"
+					);
+				}else{
```
Comment (matin-binary) [inline chartiq/development/js/standard.js:1135]: seems like it needs extra spaces around `}else{` like `} else {`

File: chartiq/development/js/standard.js [hunk 2]
```diff
@@ -1127,14 +1127,26 @@ CIQ.ChartEngine.prototype.drawingClick = function (panel, x, y) {
...
+					this.controls.crossX.classList.toString().replace(/.+/gm,
+						"stx_crosshair"
+					);
+				}
+				if(typeof this.controls.crossY.classList === 'string'){
+					this.controls.crossY.classList.replace(
+						"stx_crosshair_drawing",
+						"stx_crosshair"
+					);
+				}else{
```
Comment (matin-binary) [inline chartiq/development/js/standard.js:1145]: same here.

File: chartiq/development/js/standard.js (hunk 1 above)
Comment (maryiafrantsava-deriv) [reply chartiq/development/js/standard.js]: Fixed

File: chartiq/development/js/standard.js (hunk 2 above)
Comment (maryiafrantsava-deriv) [reply chartiq/development/js/standard.js]: fixed

**Review by matin-binary — APPROVED** [review]

**Comment by balakrishna-binary** [comment]
We should not modify `chartiq/development/js/standard.js`. We might replace the file where there is a new chartiq release.
Instead we should override the function(CIQ.ChartEngine.prototype.drawingClick) using injection. Check this PR for example - https://github.com/binary-com/SmartCharts/pull/1204/files

**Review by balakrishna-binary — APPROVED** [review]

**Review by akmal-binary — APPROVED** [review]

File: src/chartiq_injections/drawingClickChartEngine.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+export const overrideDrawingClickChartEngine = () => {
```
Comment (balakrishna-binary) [inline src/chartiq_injections/drawingClickChartEngine.js:1]: Can you modify this file to TS?

**Pull request merged**